example/fine_advection/stokes.hpp

```diff
@@ -24,9 +24,10 @@ namespace advection_example {
 using namespace parthenon::driver::prelude;
 
 template <class pack_desc_t>
-TaskStatus WeightedSumData(parthenon::CellLevel cl, parthenon::TopologicalElement te,
-                           pack_desc_t &pd, MeshData<Real> *in1, MeshData<Real> *in2,
-                           Real w1, Real w2, MeshData<Real> *out) {
+TaskStatus WeightedSumDataElement(parthenon::CellLevel cl,
```

Comment:
not for this MR, but this feels like it belongs in `interface/update.cpp` just as much as `UpdateWithFluxDivergence` does.

Reply:
agreed. The next thing I plan to do with this is going to be making the update functions in this example compatible with sparse fields, remove the old sparse example, and add sparse fields to this example. Once that is all working, I think it makes sense to move this stuff to `interface/` although it semi-duplicates some things that are in there currently so we will have to decide how to deal with that.